Sources/Core/Renderer/Mesh/FluidMeshBuilder.swift

```diff
@@ -107,9 +107,11 @@ struct FluidMeshBuilder { // TODO: Make fluid meshes look more like they do in v
     tint: Vec3f
   ) {
     let basePosition = position.relativeToChunkSection.floatVector + Vec3f(0.5, 0, 0.5)
+    var cullingFluid = cullingNeighbours
+    if neighbouringBlocks[.up]?.fluidId != fluid.id { cullingFluid.remove(.up) }
```

Comment:
Avoid single line if statements, I find it worth while (especially while skim-reading) to use the extra two lines.

Reply:
That's my bad, I rushed and forgot that you had the whole conventions document attached. I'll do a formatting commit in just a moment